model/project_ref.go

```diff
@@ -49,15 +49,17 @@ type ProjectRef struct {
 	// Identifier must be unique, but is modifiable. Used by users.
 	Identifier string `bson:"identifier" json:"identifier" yaml:"identifier"`
 
-	DisplayName            string              `bson:"display_name" json:"display_name,omitempty" yaml:"display_name"`
-	Enabled                bool                `bson:"enabled,omitempty" json:"enabled,omitempty" yaml:"enabled"`
-	Private                *bool               `bson:"private,omitempty" json:"private,omitempty" yaml:"private"`
-	Restricted             *bool               `bson:"restricted,omitempty" json:"restricted,omitempty" yaml:"restricted"`
-	Owner                  string              `bson:"owner_name" json:"owner_name" yaml:"owner"`
-	Repo                   string              `bson:"repo_name" json:"repo_name" yaml:"repo"`
-	Branch                 string              `bson:"branch_name" json:"branch_name" yaml:"branch"`
-	RemotePath             string              `bson:"remote_path" json:"remote_path" yaml:"remote_path"`
-	PatchingDisabled       *bool               `bson:"patching_disabled,omitempty" json:"patching_disabled,omitempty"`
+	DisplayName string `bson:"display_name" json:"display_name,omitempty" yaml:"display_name"`
```

Comment:
this whitespace change isn't ideal, we'd prefer to have the variable types lined up like they were before. I assume your editor auto did this, any way to tweak the settings?

Reply:
Ah okay fair. In that case can you move RemotePath above DisplayName?